src/Models/BaseElement.php

```diff
@@ -1144,11 +1173,11 @@ public function getAuthor()
      */
     public function getStyleVariant()
     {
-        $style = $this->Style;
+        $style = $this->Style ?? '';
         $styles = $this->config()->get('styles');
 
         if (isset($styles[$style])) {
-            $style = strtolower($style ?? '');
+            $style = strtolower($style);
```

Comment:
PHP 8.5 compat failure with trying to pass `$style` as an array index when it was null. Just moved `?? ''` further up.